### DISCUSSION TITLE
Updating stashapp

### DISCUSSION THREAD

**Discussion by 1Jackhammer** [stashapp/stash#5424] [Q&A] [answered]
Hi 

Firstly, thank you for a fantastic app. It's an absolute game changer!

Had a brief look in Q&A, but didn't find the answer. Pardon my ignorance, but is there a simple way to update my stashapp to the newest version and still keeping the precious database I already built?

Thanks

**Comment by feederbox826** [comment]
linux install
```
pip install --upgrade stashapp-tools
```

docker install
```
docker exec -it stash
pip install --upgrade --break-system-packages stashapp-tools
```

**Reply by 1Jackhammer** [reply]
Thanks for the fast reply!

Apologies for being unworthy here, but can you elaborate a little more on what I need to do. Apparantly i don't have docker installed as I just downloaded the stashapp package for windows and nothing else (running it on windows). 

Any help appreciated!

**Reply by feederbox826** [reply]
In that case, open powershell and run the linux command
```
pip install --upgrade stashapp-tools
```

If you don't have pip installed then download python 3.13.0 (which includes pip) from https://www.python.org/downloads/

**Comment by Maista6969** [comment, marked as answer]
Are you referring to the Stash program or the Python library? Because if it's the program itself you can just download the newest executable file from [releases](https://github.com/stashapp/stash/releases) and run that, Stash does not install itself but puts the database (and all of its other files) in the default `%USERPROFILE%/.stash` folder

**Reply by 1Jackhammer** [reply]
I guess I'm referring to the program itself. I'll give it a try, thanks!

**Reply by 1Jackhammer** [reply]
This did the trick, thanks to the both of you!

Guess I simply asked the wrong question at first... sorry.
